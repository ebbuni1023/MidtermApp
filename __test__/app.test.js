import 'react-native';
import React from 'react';
import Ads from '../src/screens/Ads'
import renderer from 'react-test-renderer'
import { AdMobBanner, AdMobRewarded, setTestDeviceIDAsync } from "expo-ads-admob";


test('App snapShot', () => {
    const snap = renderer.create(
        <Ads />
    ).toJSON();
expect(snap).toMatchSnapshot();
});