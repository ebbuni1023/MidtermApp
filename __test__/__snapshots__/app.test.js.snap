// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapShot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#ecf0f1",
      "flex": 1,
      "paddingTop": 44,
    }
  }
>
  <View
    accessibilityRole="button"
    accessibilityState={Object {}}
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "margin": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        InterstitialAd
      </Text>
    </View>
  </View>
  <View
    accessibilityRole="button"
    accessibilityState={Object {}}
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "margin": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        rewardedVideoAd
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "bottom": 0,
        "position": "absolute",
        "width": "100%",
      }
    }
  >
    <ViewManagerAdapter_ExpoAdsAdMobBannerView
      proxiedProperties={
        Object {
          "adUnitID": "ca-app-pub-3940256099942544/6300978111",
          "additionalRequestParams": Object {
            "npa": "1",
          },
          "bannerSize": "fullBanner",
          "onAdViewDidDismissScreen": undefined,
          "onAdViewDidReceiveAd": [Function],
          "onAdViewWillDismissScreen": undefined,
          "onAdViewWillLeaveApplication": undefined,
          "onAdViewWillPresentScreen": undefined,
          "onDidFailToReceiveAdWithError": [Function],
          "onSizeChange": [Function],
        }
      }
      style={Object {}}
    />
  </View>
  <View>
    <ViewManagerAdapter_ExpoAdsPublisherBannerView
      proxiedProperties={
        Object {
          "adUnitID": "ca-app-pub-3940256099942544/6300978111",
          "additionalRequestParams": Object {
            "npa": "1",
          },
          "bannerSize": "mediumRectangle",
          "onAdViewDidDismissScreen": undefined,
          "onAdViewDidReceiveAd": [Function],
          "onAdViewWillDismissScreen": undefined,
          "onAdViewWillLeaveApplication": undefined,
          "onAdViewWillPresentScreen": undefined,
          "onAdmobDispatchAppEvent": undefined,
          "onDidFailToReceiveAdWithError": [Function],
          "onSizeChange": [Function],
        }
      }
      style={Object {}}
    />
  </View>
</View>
`;
